client/data/services/QuestionDataService.js

```diff
@@ -76,6 +76,16 @@ tieData.factory('QuestionDataService', [
         return question.getTitle();
       },
       /**
+       * Returns the question's version.
+       * Currently returns null, as question versioning isn't implemented yet.
+       * TODO(eyurko): Return correct question version, once implemented.
+       *
+       * @returns {null}
+       */
+      getQuestionVersion() {
```

Comment:
Btw, I just noticed the style for this isn't quite right -- we typically do: `functionName: function(args) {`.  It looks like a couple of the functions above/below this one also need this small fix; would you mind doing that too in this PR?

Reply:
sure